pandas/io/formats/style.py

```diff
@@ -849,7 +861,7 @@ def use(self, styles):
         self._todo.extend(styles)
         return self
 
-    def set_uuid(self, uuid):
+    def set_uuid(self, uuid: str) -> "Styler":
```

Comment:
i dont actually use this code, is the uuid here going to be specifically a hex str?  if so, we should look into whether there is a more specific annotatiton for that

Reply:
I _think_ the mechanism for this would be NewType, https://mypy.readthedocs.io/en/latest/more_types.html#newtypes

Reply:
@simonjayhawkins Don't you think that's a bit of an overkill?
By overkill I mean, creating a new class just to annotate one variable in one function, just seems too much.